src/btree/bt_debug.c

```diff
@@ -970,19 +952,19 @@ __wt_debug_cursor_page(void *cursor_arg, const char *ofile)
  *     Dump the history store tree given a user cursor.
  */
 int
-__wt_debug_cursor_tree_hs(void *cursor_arg, const char *ofile)
+__wt_debug_cursor_tree_hs(void *session_arg, const char *ofile)
   WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
```

Comment:
@bvpvamsikrishna I can't replace this function with `__wt_debug_tree_all `. This function is not called in dump. It is called when we about to crash in test format. When it is called, we only have the btree of the data store. That is why we have to first open a hs cursor and then get its btree.